### DISCUSSION TITLE
Setting highlights properly

### DISCUSSION THREAD

**Discussion by enrico223** [ahrm/sioyek#688] [Q&A] [answered]
My problem is that highlights cover the letters/words highlighted just on Sioyek, no matter whether they are embedded or not. I do not have problems on other PDF viewers.
I filed an issue (#671), and I did not find any other information online or on previous issues. 
I think it could be a configuration problem, here my [config](https://github.com/enrico223/.config/tree/main/sioyek). Can someone help me to debug my problem?

**Comment by ahrm** [comment, marked as answer]
It is not a configuration problem, we currently render the highlights on top of the PDF file. At the moment I am busy porting sioyek to mobile devices but once that is done I will get back to issues.
